plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClient.java

```diff
@@ -421,6 +421,7 @@ protected void renameSchema(ConnectorSession session, Connection connection, Str
     @Override
     public void addColumn(ConnectorSession session, JdbcTableHandle handle, ColumnMetadata column)
     {
+        verify(handle.getAuthorization().isEmpty(), "authorization should be empty");
```

Comment:
I think we can skip these checks for connectors. Notice that clickhouse connector is not using authorization field so there is no way to get it.

Let's limit changes only to base-jdbc module.

Reply:
But if any of the other jdbc based connector overrides it - won't it leak in those case ?